src/bundle/ui-dev/src/modules/multi-file-upload/multi.file.upload.module.js

```diff
@@ -186,36 +142,27 @@ export default class MultiFileUploadModule extends Component {
         }));
     }
 
-    /**
-     * Prevents default event actions
-     *
-     * @method preventDefaultAction
-     * @param {Event} event
-     * @memberof MultiFileUploadModule
-     */
     preventDefaultAction(event) {
         event.preventDefault();
         event.stopPropagation();
     }
 
-    /**
-     * Renders multi file upload button,
-     * that allows to open multi file upload popup.
-     *
-     * @method renderBtn
-     * @returns {null|Element}
-     * @memberof MultiFileUploadModule
-     */
     renderBtn() {
         if (!this.props.withUploadButton) {
             return null;
         }
 
+        const Translator = getTranslator();
         const { uploadDisabled } = this.state;
         const label = Translator.trans(/*@Desc("Upload")*/ 'multi_file_upload_open_btn.label', {}, 'ibexa_multi_file_upload');
-
+        const isTriggeredBySubitems = this.props.triggeredBy === MODULES_CAN_TRIGGER_MFU_LIST['subitems'];
```

Comment:
Are we using `MODULES_CAN_TRIGGER_MFU_LIST` just to change the button CSS class? If true, in such a case, isn't it a better idea to exclude this button from the MFU module? We do not have to care about BC since the changes will be applied to v5.0. I think that the button should not be a part of MFU.

Reply:
Target branch is 4.6 so it will be BC, but like you wrote probably in v5.0 we can move this button